proposals/csharp-12.0/collection-expressions.md

```diff
@@ -98,28 +100,23 @@ Collection literals are [target-typed](https://github.com/dotnet/csharplang/blob
 ## Conversions
 [conversions]: #conversions
 
-A *collection literal conversion* allows a collection literal expression to be converted to a type.
+A *collection expression conversion* allows a collection literal expression to be converted to a type.
 
-The following implicit *collection literal conversions* exist from a collection literal expression:
+The following implicit *collection expression conversions* exist from a collection literal expression:
 
 * To a single dimensional *array type* `T[]` where:
-
```

Comment:
The blank lines were added in https://github.com/dotnet/csharplang/pull/7468, and might be necessary.

Reply:
ok :)